ai/worker/docker.go

```diff
@@ -766,12 +766,12 @@ func (m *DockerManager) GetCapacity() Capacity {
 	}
 }
 
-func (m *DockerManager) monitorInUse() {
+func (m *DockerManager) monitorInUse(pipeline string) {
 	if monitor.Enabled {
 		capacity := m.GetCapacity()
-		monitor.AIContainersInUse(capacity.ContainersInUse, "", "")
-		monitor.AIContainersIdle(capacity.ContainersIdle, "", "")
-		monitor.AIGPUsIdle(len(m.gpus) - len(m.gpuContainers)) // Indicates a misconfiguration so we should alert on this
+		monitor.AIContainersInUse(capacity.ContainersInUse, pipeline)
+		monitor.AIContainersIdle(capacity.ContainersIdle, pipeline, "")
```

Comment:
@leszko Any idea how can I get OrchestratorURI? I think we'd need to call [`suri, err := getServiceURI(n, *cfg.ServiceAddr)`](https://github.com/livepeer/go-livepeer/blob/531a04348d210b04f9ebd558ad22c9c98fcc12e1/cmd/livepeer/starter/starter.go#L1580) for `n.NodeType == core.AIWorkerNode`...